From f94ff10d5742ed92531524d3b6c2b55170c0643c Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Fri, 3 Apr 2015 18:35:18 +0800
Subject: [PATCH] MLK-10585 ASoC: imx-wm8958: disable power down time

At the end of playback and before power down time, write i2c will
failed.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 sound/soc/fsl/imx-wm8958.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/fsl/imx-wm8958.c b/sound/soc/fsl/imx-wm8958.c
index fd029e0..f0b850e 100644
--- a/sound/soc/fsl/imx-wm8958.c
+++ b/sound/soc/fsl/imx-wm8958.c
@@ -427,6 +427,7 @@ static int imx_wm8958_probe(struct platform_device *pdev)
 	data->dai.cpu_dai_name = dev_name(&cpu_pdev->dev);
 	data->dai.platform_of_node = cpu_np;
 	data->dai.ops = &imx_hifi_ops;
+	data->dai.ignore_pmdown_time = 1;
 	data->dai.dai_fmt |= SND_SOC_DAIFMT_I2S | SND_SOC_DAIFMT_NB_NF;
 
 	data->card.dev = &pdev->dev;
-- 
1.8.0

